package.json

```diff
@@ -13,19 +13,25 @@
   "scripts": {
     "start": "make dev-compose-up && concurrently \"make run\" \"livereload . -e 'html,njk'\"",
     "lint": "yarn lint:tests && yarn lint:test-schemas",
-    "lint:tests": "rm -rf ./tests/functional/generated_pages && eslint tests/functional/spec --fix",
-    "lint:test-schemas": "eslint test_schemas/*/*.json",
+    "lint:tests": "prettier --check tests/functional/**/*.js && eslint tests/functional/spec",
+    "lint:test-schemas": "prettier --check test_schemas/*/*.json && eslint test_schemas/*/*.json",
     "test_functional": "./node_modules/.bin/wdio tests/functional/wdio.conf.js",
     "generate_pages": "rm -rf ./tests/functional/generated_pages && pipenv run python -m tests.functional.generate_pages schemas/en/ ./tests/functional/generated_pages -r '../../base_pages' && pipenv run python -m tests.functional.generate_pages test_schemas/en/ ./tests/functional/generated_pages -r '../../base_pages'",
-    "format": "prettier test_schemas/*/*.json --write"
+    "format": "yarn format:tests && yarn format:test-schemas",
+    "format:tests": "prettier tests/functional/**/*.js --write && eslint --fix tests/functional/**/*.js",
```

Comment:
Why does eslint use `tests/functional/**/*.js` here and `tests/functional/spec` in the lint command?

Reply:
They should both use /**/*.js - the `/tests/functional/spec` was previously what was targeted, but there is no reason to be that restrictive